tests/driver_lsm6ds3/README.md

```diff
@@ -0,0 +1,7 @@
+# About
+Test application for the LSM6DS3 sensor, with 3D accelerometer and 3D gyroscope.
+
+# Usage
+Initialize:
+
```

Comment:
I know, this file is not elaborated well for some tests. In general I'd suggest **if** you want to provide it, just write some lines down there. Of course you can jsut test this app when the board is available